tooling/operation.js

```diff
@@ -234,8 +234,8 @@ class Operation {
    *
    * @return {array}
    */
-  getParametersAsJsonSchema() {
-    return getParametersAsJsonSchema(this.path, this.schema, this.oas);
+  getParametersAsJsonSchema(globalDefaults) {
```

Comment:
Can you add this parameter, and a short description of what it's for, into the docblock above?

Reply:
done!